framework/web/Response.php

```diff
@@ -767,7 +766,7 @@ public function xSendFile($filePath, $attachmentName = null, $options = [])
      */
     protected function getDispositionHeaderValue($disposition, $attachmentName)
     {
-        $fallbackName = str_replace('"', '\\"', str_replace(['%', '/', '\\'], '_', Inflector::transliterate($attachmentName, Inflector::TRANSLITERATE_LOOSE)));
+        $fallbackName = str_replace(array('%', '/', '\\', '"'), array('_', '_', '_', '\\"'), Inflector::transliterate($attachmentName, Inflector::TRANSLITERATE_LOOSE));
```

Comment:
For more complex replacements, how about splitting it into three lines?

```
$fallbackName = str_replace(
    ['%', '/', '\\', '"'], 
    ['_', '_', '_', '\\"'], 
    Inflector::transliterate($attachmentName, Inflector::TRANSLITERATE_LOOSE)
);
```

Reply:
Good idea =)